classes/PodsUI.php

```diff
@@ -3076,14 +3137,30 @@ public function manage( $reorder = false ) {
 										<?php esc_html_e( $filter_field['label'] ); ?>
 									</label>
 									<?php
-									echo PodsForm::field( 'filter_' . $filter, $value, 'text', $options );
+									// Prevent p div issues.
+									echo str_replace(
+										array(
+											'<div',
+											'</div>',
+										),
+										array(
+											'<span',
+											'</span>',
+										),
+										PodsForm::field( 'filter_' . $filter, $value, 'text', $options )
+									);
 								}//end if
+								?>
+									</span>
+								<?php
 							}//end foreach
 
 							if ( false !== $this->do_hook( 'filters_show_search', true ) ) {
 							?>
-								<label<?php echo ( empty( $this->filters ) ) ? ' class="screen-reader-text"' : ''; ?> for="page-search<?php esc_attr_e( $this->num ); ?>-input"><?php _e( 'Search', 'pods' ); ?>:</label>
-								<?php echo PodsForm::field( 'search' . $this->num, $this->search, 'text', array( 'attributes' => array( 'id' => 'page-search' . $this->num . '-input' ) ) ); ?>
+								<span class="pods-form-ui-filter-container pods-form-ui-filter-container-search">
+									<label<?php echo ( empty( $this->filters ) ) ? ' class="screen-reader-text"' : ''; ?> for="page-search<?php esc_attr_e( $this->num ); ?>-input"><?php _e( 'Search', 'pods' ); ?>:</label>
+									<?php echo PodsForm::field( 'search' . $this->num, $this->search, 'text', array( 'attributes' => array( 'id' => 'page-search' . $this->num . '-input' ) ) ); ?>
```

Comment:
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'PodsForm'.
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: 2bc65b9d5ea00006fd21386546d72dde</sub>